src/requirements.ts

```diff
@@ -79,9 +79,17 @@ function readJavaConfig() : string {
 function checkJavaVersion(java_home: string): Promise<number> {
     return new Promise((resolve, reject) => {
         cp.execFile(java_home + '/bin/java', ['-version'], {}, (error, stdout, stderr) => {
```

Comment:
I'll allow the quick n'dirty fix for now, but you should make it more robust, see #1